dbschema/1.tables.sql

```diff
@@ -13,7 +13,7 @@ CREATE TABLE tbl_token_transfers (
     block_hash TEXT NOT NULL,
     block_date TIMESTAMP WITHOUT TIME ZONE NOT NULL
 );
-CREATE UNIQUE INDEX tbl_token_transfers_transaction_hash_registry_address_token_id ON tbl_token_transfers (transaction_hash, registry_address, token_id);
+CREATE INDEX tbl_token_transfers_transaction_hash_registry_address_token_id ON tbl_token_transfers (transaction_hash, registry_address, token_id);
```

Comment:
we should still have a unique on something. your code above implies it should be the same as this but with the from and to added as well